Source/Shared/Storage/DiskStorage.swift

```diff
@@ -45,9 +55,14 @@ extension DiskStorage: StorageAware {
       throw StorageError.malformedFileAttributes
     }
 
+    let meta: [String: Any] = [
+      "filePath": filePath
```

Comment:
@zenangst you may like this 😄

Reply:
What are you talking about?

Reply:
@onmyway133 what do you mean? :)

Reply:
@zenangst before you had `Logger`, but with this you can get `file url` to do your Spots magic

Reply:
Oh like that